book/routing.rst

```diff
@@ -549,10 +549,11 @@ URL          Route     Parameters
 
 .. _book-routing-requirements:
 
-Adding Requirements
-~~~~~~~~~~~~~~~~~~~
+Route Requirements
+~~~~~~~~~~~~~~~~~~
```

Comment:
This makes sense, but you'll need to add a manual link above it, so that existing links to this section (i.e. `#adding-requirements` don't suddenly break:

```rst
.. adding-requirements:

Route Requirements
```